packages/framework/tests/Unit/Model/Transport/TransportPriceCalculationTest.php

```diff
@@ -67,12 +68,20 @@ public function testCalculateIndependentPrice(
         $pricingSettingMock
             ->expects($this->any())->method('getInputPriceType')
                 ->willReturn($inputPriceType);
+        $customerUserRoleProviderMock = $this->getMockBuilder(CustomerUserRoleProvider::class)
+            ->disableOriginalConstructor()
+            ->onlyMethods(['canCurrentCustomerUserSeePrices'])
+            ->getMock();
+
+        $customerUserRoleProviderMock
+            ->expects($this->any())->method('canCurrentCustomerUserSeePrices')
+                ->willReturn(true);
```

Comment:
just a note while walking by - should we create tests that verify the transport price is never free when `canCurrentCustomerUserSeePrices` returns false? Maybe I will see such tests later in this PR, or we can discuss it

Reply:
Hmm, thanks for pointing it out. It must have vanished or something. But I remember creating one. Here it is again: https://github.com/shopsys/shopsys/pull/3355/commits/c3252933eeea710e7abb5f48dddd901b0b32d081